src/cloco-client.ts

```diff
@@ -150,12 +151,27 @@ export class ClocoClient {
     private async initializeApplication(): Promise<void> {
 
       Logger.log.debug(`ClocoClient.initializeApplication: start.`);
+      let filename: string = `~/.cloco/cache/application_${this.options.application}`;
```

Comment:
colour me lazy - what's in `this.options.application` - a guid or text? I'm thinking what happens if you have >1 node client running or something of the sort.